application/client/src/app/service/session/dependencies/search/store.ts

```diff
@@ -75,6 +75,12 @@ export abstract class Store<T> extends Subscriber {
         return Array.from(this._entities.values());
     }
 
+    public getActive(): StoredEntity<T>[] {
+        return Array.from(this._entities.values()).filter((entity: T) => {
+            return (entity as any).definition.active;
```

Comment:
This Store is an abstraction `Store<T>`, and type `T` also has requirements: 
```
export type StoredEntity<T> = T & Hash & Recognizable & Updatable<EntityUpdateEvent<any, T>>;
```
The idea behind is:  preventing any type related issues. It means, applying of `any` is in conflict with this conception.
Also `Store` just doesn't know what is `active` / `inactive`. 
If we should to teach `Store` recognize it, we should add requirements, like
```
interface ActiveState {
     isActive(): boolean;
}
export abstract class Store<T extends ActiveState> extends Subscriber {
```
But this is will not work, because `Store<T>` stays  behind all stores, including `disabled`, but disabled entities doesn't have property `active`/`inactive`
Well, it means - it should be removed from here.